src/test/java/org/openrewrite/java/migrate/jakarta/JavaxInjectToJakartaInjectTest.java

```diff
@@ -56,7 +56,7 @@ void projectWithJavaxInject() {
             spec -> spec.after(actual -> assertThat(actual)
               .contains("<groupId>jakarta.inject</groupId>")
               .contains("<artifactId>jakarta.inject-api</artifactId>")
-              .containsPattern("<version>[0-9]+\\.[0-9]+\\.[0-9]+</version>")
+              .containsPattern("<version>[0-9]+\\.[0-9]+\\.[0-9]+[\\w\\W]*</version>")
```

Comment:
This looks a bit odd to me:
> \w	A word character: [a-zA-Z_0-9]
>\W	A non-word character: [^\w]

So then to me, `[\\w\\W]*` would seem equivalent to `.*`; correct?

Reply:
Fixed in https://github.com/openrewrite/rewrite-migrate-java/commit/0840ea5d1b3000c8454996ab083171ad1016aed1